cluster/gce/gci/configure-helper.sh

```diff
@@ -850,12 +875,17 @@ function start-kube-apiserver {
 
   if [[ -n "${KUBE_USER:-}" || ! -e /etc/srv/kubernetes/abac-authz-policy.jsonl ]]; then
     local -r abac_policy_json="${src_dir}/abac-authz-policy.jsonl"
-    remove-salt-config-comments "${abac_policy_json}"
-    if [[ -n "${KUBE_USER:-}" ]]; then
-      sed -i -e "s/{{kube_user}}/${KUBE_USER}/g" "${abac_policy_json}"
-    else
-      sed -i -e "/{{kube_user}}/d" "${abac_policy_json}"
-    fi
+    # the abac policy for these runs is different since we have RBAC enabled too.  This starts
+    # removing the unnecessary bits
+    # We need system:nodes added to the kubelet cert and then that permission is ready to be removed.
+    # We need to update the ingress and node-problem-detector tests and then we can tighten the SA permissions
+    cat > "${abac_policy_json}" <<EOF
```

Comment:
> I'm not sure how (whether) this will affect upgrades. Why do you want the abac policy specified here instead of in the checked in file?

I wasn't sure who all depended on that file.  I'm ok changing it in place if you're confident about its scope.